### PR TITLE
Update for v0.3.1

### PR DESCRIPTION
**Pull request by gtauzin**

### CODE REVIEW

File: RELEASE.rst [hunk 1]
```diff
@@ -1,3 +1,31 @@
+Release 0.3.1
+==============
+
+Major Features and Improvements
```
Comment (ulupo) [inline RELEASE.rst:4]: I guess this empy section is deliberate. Personally, I'd remove it (the release number should suggest nothing major has happened), or write "None" in the description.

**Comment by ulupo** [comment]
Great. I assume you saw my comment above. Not asking to change, so feel free to merge as things are.

**Comment by gtauzin** [comment]
> Great. I assume you saw my comment above. Not asking to change, so feel free to merge as things are.

It's no longer empty. I have updated the RELEASE.rst file.

**Comment by ulupo** [comment]
Oh yes, sorry I missed it!

**Pull request merged**